app/src/main/java/com/andrew/tasky/presentation/agenda/AgendaFragment.kt

```diff
@@ -276,4 +278,53 @@ class AgendaFragment : Fragment(R.layout.fragment_agenda) {
                 .setCardBackgroundColor(Color.parseColor("#FFFFFFFF"))
         }
     }
+
+    //Changes currentDateSelectedTextView to yesterday, today, tomorrow, or formatted date
+    private fun setupCurrentDateSelectedTextView(dateSelected: LocalDate){
+        val yesterdayDate = currentDate.minusDays(1)
+        val tomorrowDate = currentDate.plusDays(1)
+
+        if (dateSelected == yesterdayDate){
+            fragmentAgendaBinding.currentDateSelectedTextView.text = getString(R.string.yesterday)
+        }
+        else if (dateSelected == currentDate){
+            fragmentAgendaBinding.currentDateSelectedTextView.text = getString(R.string.today)
+        }
+        else if (dateSelected == tomorrowDate){
+            fragmentAgendaBinding.currentDateSelectedTextView.text = getString(R.string.tomorrow)
+        }
+        else {
+            fragmentAgendaBinding.currentDateSelectedTextView.text = dateSelected
+                .format(DateTimeFormatter.ofPattern("MMM dd yyyy"))
+        }
+    }
+
+    private fun setupAgendaItemListRecyclerView(dateSelected: LocalDate){
+        val agendaItemListSorted = mutableListOf<AgendaItem>()
+        agendaItemList.sortedBy{ agendaItem -> agendaItem.startDateAndTime }
+            .forEach{if (it.startDateAndTime.toLocalDate() == dateSelected){
+                agendaItemListSorted.add(it)}}
```

Comment:
Sorting a list based on a requirement is business logic and should therefore go into the domain layer. That makes it easier to test, since you don't need to run an emulator for the automated test (which you would need if the sorting logic was in the UI)